### DISCUSSION TITLE
Zigbee2MQTT Can not create new Domoticz device: maximum of 255 devices is reached

### DISCUSSION THREAD

**Discussion by McGyver64** [Koenkk/zigbee2mqtt#9361] [Devices]
I use Domoticz and Zigbee2MQTT for 1,5 year now and all is working well
Tried to add an extra Aqara Temp sensor (WSDCGQ11LM) but the second one i add does not show his devices 

<img width="1177" alt="Schermafbeelding 2021-10-27 om 15 13 06" src="https://user-images.githubusercontent.com/69874589/139072731-f413bc77-1145-4982-8112-4b7a81e6cb9f.png">

(Temperature + Humidity) 
(Link Quality) 
(Battery Voltage)
(Battery)
(Temperature) 
(Humidity)
(Pressure)
(Barometer) 
(Temperature + Humidity + Barometer)

I have a total of 30 (real) devices , but some devices have 6-10 different sub-devices (temperature, pressure, etc) 

<img width="1178" alt="Schermafbeelding 2021-10-27 om 15 08 07" src="https://user-images.githubusercontent.com/69874589/139072287-0b4bbb30-7285-4c51-9c97-d98d30aae3c5.png">

When i look into log file i see message
Error: Zigbee2MQTT: (Zigbee2MQTT) Can not create new Domoticz device: maximum of 255 devices is reached.

When i remove the 2 newly added sensors and i count the number of devices, i still have 286 devices and they all work fine (but it s mora as 255?)
When i delete the unused devices i am left with a total of 160 devices, but after some time, or reboot all the unused devices are back again and i still cannot add a device.

Is there a way to overcome the 255 limit or to disable the unused devices so i can add more sensors?

**Comment by Koenkk** [comment]
I suggest create an issue here: https://github.com/stas-demydiuk/domoticz-zigbee2mqtt-plugin/issues , this issue is related to domitcz not zigbee2mqtt.

**Comment by McGyver64** [comment]
But it clearly says that Zigbee2MQTT can not create......... and not Domoticz........
I dont understand where the 255 limit is comming form, it seems Zigbee2MQTT.....???

**Reply by Koenkk** [reply]
Not from z2m,maybe from the plugin, I suggest creating an issue here; https://github.com/stas-demydiuk/domoticz-zigbee2mqtt-plugin/issues

**Reply by McGyver64** [reply]
I created a new issue like suggested.
https://github.com/stas-demydiuk/domoticz-zigbee2mqtt-plugin/issues/731
